### PR TITLE
Displaying synonyms now

### LINKED ISSUE #8: Come up with a way to display "Synonyms" frontmatter when it is defined for a term
We have an optitional [hugo front matter](https://gohugo.io/content-management/front-matter/) variable of `synonyms` available. When a term has synonyms the author can write them in to this as an array, see [Command-Line Interface](https://github.com/do-community/developer-glossary/blob/main/content/terms/command-line-interface.md) example:

image

Update the [`single.html`](https://github.com/do-community/developer-glossary/blob/main/layouts/terms/single.html) template to render this out somewhere on the page.

### PR DESCRIPTION
**Pull request by InfiniteAnts**
Fixes #8 
Displays Synonyms in the single.html page now.

### CODE REVIEW

File: layouts/terms/single.html [hunk 1]
```diff
@@ -15,9 +15,16 @@ <h1>{{ .Title }}</h1>
 
       {{ .Content }}
 
+      {{ if isset .Params "synonyms" }}
+        <h2>Synonyms</h2>
+        <li>
+        <ul>{{ delimit .Params.synonyms ", " ", and " }}</ul>
```
Comment (ruf-io) [inline layouts/terms/single.html]: You've got `<ul>` and `<li>` reversed here.

**Review by ruf-io — CHANGES_REQUESTED** [review]
switch those `<ul>` and `<li>` tags and this should be good to go.

File: layouts/terms/single.html (hunk 1 above)
Comment (InfiniteAnts) [reply layouts/terms/single.html]: Oh crap. Brainfart.

Comment (InfiniteAnts) [reply layouts/terms/single.html]: Fixed it.

**Review by ruf-io — APPROVED** [review]
LGTM

**Pull request merged**